### ISSUE TITLE
Female Focused Symptoms/ Conditions Updating 1 

### ISSUE THREAD

**Issue by Lucy-WhiteSwan** [obophenotype/human-phenotype-ontology#7511]
**Preferred term label:**
mastalgia

**Synonyms**
painful breast, breast pain

**Definition (free text, please give PubMed ID)**
Pubmed ID: 16533457
Mastalgia is a medical term used to describe breast pain.

**Parent term (use hpo.jax.org/app)**
Pain HP:0012531

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
N/A

**Comment by pnrobinson** [comment]
added as HP:0034265
